pkg/statsd/dispatcher_worker.go

```diff
@@ -38,3 +41,15 @@ func (w *worker) executeProcess(cmd *processCommand) {
 	defer cmd.done() // Done with the process command
 	cmd.f(w.id, w.aggr)
 }
+
+func (w *worker) runMetrics(ctx context.Context, s statser.Statser) {
+	csw := statser.NewChannelStatsWatcher(
+		s,
+		"dispatch_aggregator",
```

Comment:
Intent of the naming is "sender_receiver".  We'll see how that pans out if I add more channels.